<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MissPutoutDlg</class>
 <widget class="QDialog" name="MissPutoutDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>474</width>
    <height>318</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>9</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>出库</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="2">
    <widget class="QLabel" name="lblPutoutDate">
     <property name="text">
      <string>出库时间:</string>
     </property>
     <property name="buddy">
      <cstring>dtePutoutDate</cstring>
     </property>
    </widget>
   </item>
   <item row="5" column="2" rowspan="3" colspan="2">
    <widget class="QWidget" name="widgetLoan" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="spacing">
       <number>3</number>
      </property>
      <property name="sizeConstraint">
       <enum>QLayout::SetMinAndMaxSize</enum>
      </property>
      <property name="margin">
       <number>3</number>
      </property>
      <item>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="1">
         <widget class="QComboBox" name="combEiName">
          <property name="editable">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="lblFiName">
          <property name="text">
           <string>最终用户:</string>
          </property>
          <property name="buddy">
           <cstring>combFiName</cstring>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QComboBox" name="combFiName">
          <property name="editable">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QDateEdit" name="dedtPlanDate">
          <property name="displayFormat">
           <string>yyyy/MM/dd</string>
          </property>
          <property name="calendarPopup">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="lblEiName">
          <property name="text">
           <string>借用人:</string>
          </property>
          <property name="buddy">
           <cstring>combEiName</cstring>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="lblPlanDate">
          <property name="text">
           <string>预定归还日期:</string>
          </property>
          <property name="buddy">
           <cstring>dedtPlanDate</cstring>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="2">
    <widget class="QLabel" name="lblPdiType">
     <property name="text">
      <string>样品标记:</string>
     </property>
     <property name="buddy">
      <cstring>combPdiType</cstring>
     </property>
    </widget>
   </item>
   <item row="4" column="3">
    <widget class="QComboBox" name="combReason">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <item>
      <property name="text">
       <string>销售</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>外借</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="4" column="2">
    <widget class="QLabel" name="lblReason">
     <property name="text">
      <string>出库原因:</string>
     </property>
     <property name="buddy">
      <cstring>combReason</cstring>
     </property>
    </widget>
   </item>
   <item row="0" column="3">
    <widget class="QCheckBox" name="cboxPutoutNow">
     <property name="text">
      <string>现在</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="3">
    <widget class="QDateTimeEdit" name="dtePutoutDate">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="displayFormat">
      <string>yyyy/MM/dd HH:mm:ss</string>
     </property>
     <property name="calendarPopup">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="2" column="3">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QComboBox" name="combPdiType">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <item>
        <property name="text">
         <string>正品</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>样品</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnOfficialChange">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>正转样</string>
       </property>
       <property name="autoDefault">
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="lblShelf">
     <property name="text">
      <string>货架号:</string>
     </property>
     <property name="buddy">
      <cstring>edtShelf</cstring>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QLineEdit" name="edtShelf">
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="8" column="0" colspan="4">
    <layout class="QHBoxLayout">
     <property name="spacing">
      <number>6</number>
     </property>
     <property name="margin">
      <number>0</number>
     </property>
     <item>
      <widget class="QPushButton" name="btnOK">
       <property name="text">
        <string>保存并关闭</string>
       </property>
       <property name="autoDefault">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <spacer>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Expanding</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>120</width>
         <height>31</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btnSaveAndConti">
       <property name="text">
        <string>保存并继续</string>
       </property>
       <property name="autoDefault">
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="6" column="1">
    <widget class="QPlainTextEdit" name="pedtPdiDesc">
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="5" column="1">
    <widget class="QPlainTextEdit" name="pedtPiDesc">
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="6" column="0">
    <widget class="QLabel" name="lblPdiDesc">
     <property name="text">
      <string>备注:</string>
     </property>
     <property name="buddy">
      <cstring>pedtPdiDesc</cstring>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="lblPiDesc">
     <property name="text">
      <string>产品描述:</string>
     </property>
     <property name="buddy">
      <cstring>pedtPiDesc</cstring>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="lblProdName">
     <property name="text">
      <string>产品型号:</string>
     </property>
     <property name="buddy">
      <cstring>edtProdName</cstring>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QLineEdit" name="edtProdName">
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="lblSerial">
     <property name="text">
      <string>序列号:</string>
     </property>
     <property name="buddy">
      <cstring>edtSerial</cstring>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QLineEdit" name="edtSerial"/>
   </item>
   <item row="0" column="0" colspan="2">
    <widget class="QLabel" name="lblTips">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>edtProdName</tabstop>
  <tabstop>edtSerial</tabstop>
  <tabstop>edtShelf</tabstop>
  <tabstop>pedtPiDesc</tabstop>
  <tabstop>pedtPdiDesc</tabstop>
  <tabstop>combReason</tabstop>
  <tabstop>combEiName</tabstop>
  <tabstop>combFiName</tabstop>
  <tabstop>dedtPlanDate</tabstop>
  <tabstop>cboxPutoutNow</tabstop>
  <tabstop>dtePutoutDate</tabstop>
  <tabstop>combPdiType</tabstop>
  <tabstop>btnOfficialChange</tabstop>
  <tabstop>btnSaveAndConti</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>combReason</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>MissPutoutDlg</receiver>
   <slot>s_outReasonChanged(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>363</x>
     <y>95</y>
    </hint>
    <hint type="destinationlabel">
     <x>376</x>
     <y>1</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cboxPutoutNow</sender>
   <signal>clicked(bool)</signal>
   <receiver>dtePutoutDate</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>363</x>
     <y>24</y>
    </hint>
    <hint type="destinationlabel">
     <x>363</x>
     <y>35</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnOK</sender>
   <signal>clicked()</signal>
   <receiver>MissPutoutDlg</receiver>
   <slot>s_saveAndClose()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>72</x>
     <y>303</y>
    </hint>
    <hint type="destinationlabel">
     <x>9</x>
     <y>321</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnSaveAndConti</sender>
   <signal>clicked()</signal>
   <receiver>MissPutoutDlg</receiver>
   <slot>s_saveAndConti()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>463</x>
     <y>303</y>
    </hint>
    <hint type="destinationlabel">
     <x>378</x>
     <y>325</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>edtSerial</sender>
   <signal>editingFinished()</signal>
   <receiver>MissPutoutDlg</receiver>
   <slot>s_checkSerial()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>209</x>
     <y>80</y>
    </hint>
    <hint type="destinationlabel">
     <x>127</x>
     <y>22</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnOfficialChange</sender>
   <signal>clicked()</signal>
   <receiver>MissPutoutDlg</receiver>
   <slot>s_officialChange()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>412</x>
     <y>74</y>
    </hint>
    <hint type="destinationlabel">
     <x>380</x>
     <y>66</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>s_outReasonChanged(int)</slot>
  <slot>s_saveAndConti()</slot>
  <slot>s_saveAndClose()</slot>
  <slot>s_checkSerial()</slot>
  <slot>s_officialChange()</slot>
 </slots>
</ui>
